examples/notebooks/show_remote_cubes.py

```diff
@@ -15,7 +15,8 @@ def show_remote_cubes(bucket, endpoint_url, region_name):
 
     cube_names = []
     df = pd.DataFrame(
-        columns=['cube_name', 'chunksize (time, lat, lon)', 'var_nums', 'variables', 'start_date', 'end_date'])
+        columns=['cube_name', 'chunks (time, lat, lon)', 'number_of_variables', 'variables',
```

Comment:
`'chunks (time, lat, lon)'` still contains spaces, i.e. characters that are not valid for Python identifiers.